debugstick.sc

```diff
@@ -12,17 +12,73 @@ __config() -> {
             'source' -> 'https://raw.githubusercontent.com/Arcensoth/mcdata/master/processed/reports/blocks/simplified/data.json',
             'target' -> 'data.json',
         }
-    ]
+    ],
+    'commands' -> {
+        '' -> _() -> _give_debugstick(),
+        'blacklistStates add <stateToAdd>' -> _(stateToAdd) -> _add_blacklisted_state(stateToAdd),
+        'blacklistStates remove <stateToRemove>' -> _(stateToRemove) -> _remove_blacklisted_state(stateToRemove)
+    },
+    'arguments' -> {
+        'stateToAdd' -> {'type' -> 'term', 'suggester' -> _(args) -> (global_possible_states)},
+        'stateToRemove' -> {'type' -> 'term', 'suggester' -> _(args) -> (global_blacklist_states)}
+    }
 };
 
 global_properties = read_file('data','json');
-global_blacklist_states = ['waterlogged'];
+global_possible_states = [];
 
-for(global_properties,
+_give_debugstick() -> if(player() ~ 'permission_level' >= 2, spawn('item', player()~'pos', '{PickupDelay:0,Item:{id:"minecraft:debug_stick",Count:1b}}'););
```

Comment:
Aggiungerei la tag owner all'item per impedire che venga raccolto da altri player e darei un feedback con un messaggio in chat...